### PR TITLE
meta(isort): Move isort config to setup.cfg

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -24,3 +24,12 @@ max-line-length = 100
 ignore = W690,E701,E70,E711,E721
 aggressive = 1
 exclude = */south_migrations/*
+
+[isort]
+line_length=100
```
Comment (EvanPurkhiser) [inline setup.cfg]: pep8 line-length above is 100

File: setup.cfg [hunk 2]
```diff
@@ -24,3 +24,12 @@ max-line-length = 100
...
+lines_between_sections=1
+multi_line_output=5
+known_first_party=sentry
+default_section=THIRDPARTY
```
Comment (EvanPurkhiser) [inline setup.cfg]: Ensure's unknown modules end up in the 3rd party section

File: requirements-dev.txt [hunk 3]
```diff
@@ -2,4 +2,4 @@ autopep8
 Babel
 flake8>=2.6,<2.7
 pycodestyle>=2.0,<2.1
-isort>=4.2.2,<4.3.0
+isort>=4.3.4
```
Comment (JoshuaRLi) [inline requirements-dev.txt]: Can you minor-pin this? `isort>=4.3.4,<4.4.0`

**Comment by HazAT** [comment]
Can we either merge or close this? 
I am not really sure what this PR is about but I get a daily notification in slack for this 😅

**Review by dcramer — APPROVED** [review]

**Comment by markstory** [comment]
I've fixed the conflicts and will get this merged once CI passes.

File: requirements-dev.txt [hunk 4]
```diff
@@ -1,6 +1,6 @@
 autopep8>=1.3.5,<1.4.0
 Babel
 flake8>=3.5.0<3.6.0
-isort>=4.2.2,<4.3.0
+isort>=4.3.4,<4.4.0
 pycodestyle>=2.3.1,<2.4.0
-sentry-flake8>=0.0.1
+sentry-flake8>=0.0.1
```
Comment (EvanPurkhiser) [inline requirements-dev.txt]: We lost our linebreak here !

Comment (markstory) [reply requirements-dev.txt]: GitHub online editor for the lose. I'll put the newline back.

**Pull request merged**